docs/authoring.md

```diff
@@ -86,78 +86,140 @@ To make your component available as a NuGet package, it is important to include
+                <activatableClass name="MyAuthoredComponent.Class2"
+                                  threadingModel="both"
+                                  xmlns="urn:schemas-microsoft-com:winrt.v1" />
+              </file>
+            </assembly>
+            ```
+
+      3. Modify the project to include the manifest file in the output when deploying the project. Right-click on the file in **Solution Explorer**, select **Properties**, and set the **Content** property to **True** using the drop-down arrow on the right.
+
+### Windows App SDK Applications
+
+Authoring and consuming WinUI types in Windows App SDK applications is in development and is supported with some exceptions.
+Component authors can implement WinUI types using the Windows App SDK C# class library project template. 
+Components can be consumed by application projects that reference the Windows App SDK.
 
-### Consuming from packaged applications
+#### Support Matrix
 
-Consuming C#/WinRT components from MSIX-packaged applications is supported with a few modifications.
+The following table outlines the Windows App SDK application types that are supported when consuming C#/WinRT authored components implementing WinUI types:
+
+- ✅ means the scenario works
+- 🟨 means the scenario partially works (i.e. with specific limitations)
+- 🟥 means the scenario does not work
+- Note: "WAP" refers to the usage of a separate Windows Application Packaging Project in packaged apps.
+
+##### C++
+|Application Type|Project or Package Reference|Supported?|Tracking issue|
+|:--|:-:|:-:|:-:|
+|C++ Packaged|Project Reference| ✅ | |
+|C++ Packaged with WAP |Project Reference| 🟥 | [#1120](https://github.com/microsoft/CsWinRT/issues/1120) |
+|C++ Unpackaged|Project Reference| 🟥 | [#1116](https://github.com/microsoft/CsWinRT/issues/1116) |
+|C++ Packaged|Package Reference|🟨 | [#1118](https://github.com/microsoft/CsWinRT/issues/1118) |
+|C++ Packaged with WAP |Package Reference| 🟥 | [#1123](https://github.com/microsoft/CsWinRT/issues/1123) |
+|C++ Unpackaged|Package Reference|🟨 | [#1118](https://github.com/microsoft/CsWinRT/issues/1118) |
+
+##### C#
+|Application Type|Project or Package Reference|Supported?|Tracking issue|
+|:--|:-:|:-:|:-:|
+|C# Packaged|Project Reference| ✅ |
+|C# Packaged with WAP|Project Reference| ✅ |
+|C# Unpackaged|Project Reference|🟥| [#1116](https://github.com/microsoft/CsWinRT/issues/1116) |
+|C# Packaged|Package Reference| 🟨 | [#1118](https://github.com/microsoft/CsWinRT/issues/1118) |
+|C# Packaged with WAP |Package Reference| 🟨 | [#1118](https://github.com/microsoft/CsWinRT/issues/1118) |
+|C# Unpackaged|Package Reference|🟥 | [#1116](https://github.com/microsoft/CsWinRT/issues/1116) |
+
+#### Consumption of WinUI controls from C++ apps
+
+If a C++ app is consuming a component that provides a custom user control, there are a few extra configuration steps:
+
+1. In the app's `pch.h` header file, add the following line:

```

Comment:
Make note somewhere MyAuthoredComponent should be renamed to their component name.